senior_teacher.rb

```diff
@@ -1,50 +1,12 @@
-class SeniorTeacher
-  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
-  attr_accessor :name
+require_relative 'teacher'
 
-  def initialize(options={})
-    @phase = 3
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 1000
-  end
-
-  def offer_high_five
-    "High five!"
-  end
+class SeniorTeacher < Teacher
+  attr_reader :performance_rating
+  RATING_LIMIT = 90
+  TEACH_STUFF_TEXT = ["Listen, class, this is how everything works, fo SHO! ", "*drops flat-out insane knowledge bomb* ", "... You're welcome. *saunters away*" ]
 
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
-
-  def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works, fo SHO! "
-    response += "*drops flat-out insane knowledge bomb* "
-    response += "... You're welcome. *saunters away*"
-    response
-  end
-
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
-  def set_performance_rating(rating)
-    response = ""
-    if rating > 90
-      response = "Yay, I'm a great employee!"
-      receive_raise(@target_raise)
-    else
-      response += "Oh, well -- thanks to this actionable, specific, and kind "
-      response += "feedback, I'll do better next time."
-    end
-    response
+  def initialize(options={})
+    super
```

Comment:
If the only thing you're doing in your method is call up to super, might as well remove the method from your class altogether.